src/ccalls.jl

```diff
@@ -68,11 +68,14 @@ GAP: [ 1 ]
 """
 function evalstr(cmd::String)
     res = evalstr_ex(cmd * ";")
-    res = res[end]
-    if res[1] == false
+    if any(x->x[1] == false, res)
       # error
-      GAP.error_handler()
-    elseif Globals.ISB_LIST(res, 2)
+      global last_error
+      error_handler()
```

Comment:
This is required in some cases (e.g. for `g"\\"`) to get the error; yet it is actively bad in other cases (e.g. for `GAP.julia_to_gap(1:2 ^ 62)` resp. `GAP.evalstr("[1..4611686018427387904]")` where it swallows the already printed error.

Also, if the string contains multiple commands leading to (syntax) errors, in general we only see one of them. Fixing this will require improving the GAP kernel functions `READ_ALL_COMMANDS` resp. `GAP_EvalString`. I've begun work on this, but I hope we find a quick&dirty fix for this for now so that we can move on w/o rebuilding the JLL *yet again* (e.g. by putting relevant code into JuliaInterface)